ReactiveCocoaTests/Swift/SignalSpec.swift

```diff
@@ -658,6 +658,48 @@ class SignalSpec: QuickSpec {
 				expect(values).to(equal([ true, false, true ]))
 			}
 
+			it("should skip duplicate optional values") {
+				let (baseSignal, observer) = Signal<Bool?, NoError>.pipe()
+				let signal = baseSignal.skipRepeats()
+
+				var values: [Bool?] = []
+				signal.observeNext { values.append($0) }
+
+				expect(values.count).to(equal(0))
+
+				observer.sendNext(true)
+				expect(values.count).to(equal(1))
+				expect(values[0]).to(equal(true))
```

Comment:
Unfortunately its not possible to do
```
expect(values).to(equal([true]))
```

because arrays with optional values are not equatable

:exclamation: `error: binary operator '==' cannot be applied to two '[Bool?]' operands`